actions/helm_generate.go

```diff
@@ -0,0 +1,39 @@
+package actions
+
+import (
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+type nopWriteCloser struct {
+	io.Writer
+}
+
+func (n nopWriteCloser) Close() error {
+	return nil
+}
+
+// NopWriteCloser returns a WriteCloser with a no-op Close method wrapping
+// the provided Writer w.
+func NopWriteCloser(w io.Writer) io.WriteCloser {
+	return nopWriteCloser{w}
+}
+
+func generateParams(stage bool, fs fileSys, whereTo string, paramsComponentMap genParamsComponentMap) error {
+	var executeTo = NopWriteCloser(os.Stdout)
+	if stage {
```

Comment:
make sure to `defer executeTo.Close()` here. if it ends up being a file, you'll want to close it